cmd/boot-script-service/routers.go

```diff
@@ -211,3 +215,12 @@ func endpointHistoryGet(w http.ResponseWriter, r *http.Request) {
 		sendAllowable(w, "GET")
 	}
 }
+
+func notifyTarget(url string, data string) {
```

Comment:
I think this needs to check the response code and potentially retry a few times.  A fire-and-forget POST could easily get missed.  At the very least we should log any IPs we weren't able to notify for.

Reply:
Sure! Thoughts on just having a hard-coded retry count for now? Also, for logs, is there a particular format we use for error logging? I see several calls to `base.SendProblemDetailsGeneric()`, so presumably that's the helper function to use here.